covid19-etl/etl/jhu_to_s3.py

```diff
@@ -566,7 +566,11 @@ def nested_dict_to_geojson(self):
                         del feat_county["properties"]["date"]
                         features.append(feat_county)
 
-        geojson = {"type": "FeatureCollection", "features": features}
+        geojson = {
+            "type": "FeatureCollection",
+            "features": features,
+            "last_updated": self.latest_date,
```

Comment:
The IL map only uses `jhuJsonByLevelLatest` == `jhu_json_by_level_latest.json` == `JSON_BY_LEVEL_FILENAME`.
The global map uses both.
Since we want to add the date to the `ControlPanel` for both maps, I would suggest adding this field to `jhu_json_by_level_latest.json` instead of `jhu_geojson_latest.json`. This way we can also avoid adding a custom field that's not valid geojson to the geojson file

Regardless of which file you add it to, could you add the field in the comment describing the file format at the top of `jhu_to_s3.py`